### ISSUE TITLE
Apple // Bejeweled crash with version 1.26.2.3

### ISSUE THREAD

**Issue by MarcoVerpelli** [AppleWin/AppleWin#422]
I decided to open a github account ...
It's not a serious problem but it could hide something more serious.

![a2bejwld_000000000](https://cloud.githubusercontent.com/assets/28804429/26244124/cdf2e6ea-3c8e-11e7-8a67-92478f2a6074.png)

**Comment by tomcw** [comment]
Thanks for reporting this.

I reproduced (AppleWin 1.26.2.1 and HEAD) with the mouse-card enabled. Removing this mouse-card from slot-4 allows the game to boot and play fine. The image is [here](http://mirrors.apple2.org.za/ftp.apple.asimov.net/images/games/misc/a2bejwld.dsk).

I tried with AppleWin 1.25.0.3 and it doesn't crash - but the mouse doesn't appear to work with the game (even though it claims to support the mouse).

**Comment by tomcw** [comment]
Regression from commit 5470feb, specifically this line added to `MemSetPaging()`:
```
				IO_SELECT_InternalROM = 0;	// GH#392
```

**Comment by tomcw** [comment]
See #423 - only a $CFFF access or an MMU reset can reset INTC8ROM (which AppleWin used to call IO_SELECT_InternalROM).

**Comment by tomcw** [comment]
I traced this crash on AppleWin 1.26.2.3:

The 6502 is executing code at $CCE0 (internal ROM):
```
CCE0:91 28     STA $(28),Y
```

A mousecard interrupt (VBl) occurs, and the code vectors to $C3FA:
```
C3FA:2C 15 C0  BIT $C015
C3FD:8D 07 C0  STA $C007
``` 
At the end of the interrupt handler the code returns to execute the floating bus:
NB. The `g` or `gg` MODE_STEPPING commands will now break on PC reading floating bus or I/O memory.
```
01E5:8D 06 C0  STA $C006	; INTCXROM OFF = Enable slot ROM from $C100-$C7FF
01E8:40        RTI 
CCE2:00        BRK 		; PC reads from floating bus or I/O memory
```

The problem is that INTC8ROM is active, but isn't being honoured:

The `STA $C006` enables the slot rom at $C100..C7FF, but because INTC8ROM is active, then internal ROM must remain present at $C800..CFFF. If INTC8ROM were implemented correctly, then the RTI would return to  internal ROM.

Anyway this is fixed in #423.
